zeebe/topology/src/main/java/io/camunda/zeebe/topology/state/ClusterTopology.java

```diff
@@ -243,6 +243,20 @@ public int partitionCount() {
         members.values().stream().flatMap(m -> m.partitions().keySet().stream()).distinct().count();
   }
 
+  public Integer replicationFactor() {
```

Comment:
:upside_down_face: I'd rename to `minReplicationFactor()` to make it clear that the replication factor can be different for each partition.